docs/qa/test_engineering

```diff
@@ -0,0 +1,44 @@
+Normandy Test Engineering
+=========================
+This is the repository for all information related to test engineering for the Normandy project.  This will outline our resources and test process for releases.
+
+The primary focus of testing will be on the Firefox client and the API server. The Normandy Firefox client follows the release process and train schedule for the Firefox browser. The Normandy server is tested using the API server and recipes.
+
+The Delivery Console is not currently being tested, as it’s a much lower priority. 
+
+The test strategy incorporates manual and automated testing. Different approaches are outlined for each area. 
+
+Normandy Server
+--------------
+The Normandy Server is an API server with recipes. This feature is high priority, and needs to maintain a high level of service. There are two versions with different functionality. V1 is the server clients connect to. V3 is the server that Admins connect to.
+
+- v1
+Automation:
+Test Engineer: chartjes
+Status:  In process
```

Comment:
I suspect the sections like this will probably need to be reworked to be valid reStrucuturedText document. Sphinx has docs about RST here: http://www.sphinx-doc.org/en/master/usage/restructuredtext/index.html

Reply:
I reviewed that page and am not sure what needs to be done for formatting as it was intended to be just a list of info.

Reply:
You can see [in this Github preview of the document](https://github.com/mozilla/normandy/pull/1793/files?short_path=99599b2#diff-99599b2d5a06775a6b1849ab92de1c6a) that it is currently rendering everything on a single line, with the "v1" and "v3" as interspersed bullet points. That's not very readable.